### PR TITLE
Don't allow "local" timezone. Replace Timex w. Calendar.

### PR DESCRIPTION
**Pull request by lau**
Depending on a "local" timezone global variable is a bad idea. See more here: http://www.creativedeletion.com/2015/08/07/why-not-to-use-server-local-time.html

### CODE REVIEW

**Comment by c-rack** [comment]
@lau Thank you for your contribution. I see several issues with this PR, but maybe you can clarify on them:

1. Your pull request does not follow our contribution schema, which requests a "Problem/Solution" statement (see commit history).
2. You want remove a valid and working option from a project with over 3,000 downloads per week without prior discussion on the possible effects for existing user base.
3. You want to change the Timex dependency to Calendar without giving a reason why this should be changed. You also do not reveal that you are the author of Calendar, so this pull request looks like an attempt to just make your library more popular, which would not be a valid problem for us to solve.

**Comment by princemaple** [comment]
@c-rack FYI the PR description is a valid problem. (Not sure if it was added after your comment)
@lau is a nice guy who loves to fix all the wrong implementations of datetime on the internet :) I'm sure it's not the case that he did this for the sake of popularizing his own package.

File: lib/quantum/executor.ex [hunk 1]
```diff
@@ -2,22 +2,21 @@ defmodule Quantum.Executor do
...
       tz0    -> tz0
     end
 
     case Application.get_env(:quantum, :timezone, :utc) do
-      :utc   -> t |> Timex.to_datetime |> Timezone.convert(tz_final)
-      :local -> t |> Timex.to_datetime(:local) |> Timezone.convert(tz_final)
-      tz     ->
-        t |> Timex.to_datetime(tz_final)
+      :utc   -> dt |> Calendar.DateTime.from_erl!("Etc/UTC") |> Calendar.DateTime.shift_zone!(tz_final)
+      :local -> raise "error - local timezone not allowed"
```
Comment (princemaple) [inline lib/quantum/executor.ex]: FYI this is the line that drops the coverage.

**Comment by lau** [comment]
> @lau Thank you for your contribution.

You're welcome. Thanks for making and maintaining this library.

1) Sorry, I didn't see your contribution schema.

2) I posted a link to a blog post that explains why it is a bad idea to use the "local" timezone for the kind of things that cron jobs do. In order to remove the "local" option, there could be a major version bump. In a combination with deprecation warnings, this would help users transition to either a specific time zone or UTC.

3) LOL. Actually I just wanted to reduce the number of dependencies, since I'm involved in a project that will probably use quantum-elixir. I hope the Calendar package won't be needed in the future. Maybe with a future version of Elixir I'll make a PR that uses just built in Elixir functions + tzdata*, but unfortunately that isn't feasible with Elixir 1.3.

It was a good decision to make UTC the default. But having the "local" option hurts more than it benefits IMHO. That's why I'm suggesting to remove it.

*) full disclosure 😄  : I'm the author of tzdata that quantum_elixir currently has as a dependency.

**Comment by c-rack** [comment]
Thanks for clarification and my apologies for assuming self-promotion! Looking forward to your next pull requests. Having a Elixir-only version (plus tzdata of course) would be wonderful.

**Pull request merged**